Assets/MixedRealityToolkit/Inspectors/MixedRealityShaderGUI.cs

```diff
@@ -16,25 +16,25 @@ public abstract class MixedRealityShaderGUI : ShaderGUI
     {
         protected enum RenderingMode
         {
-            Opaque,
-            TransparentCutout,
-            Transparent,
-            PremultipliedTransparent,
-            Additive,
-            Custom
+            Opaque = 0,
```

Comment:
Does Unity provide any enums for theses values we could use, to always stay in sync and know when we're not?

Reply:
They do have these enums, but they are in an internal class. ☹️

Reply:
😢